src/trace.ts

```diff
@@ -30,7 +30,7 @@ export interface TraceSpan {
   kind: SpanKind;
   name: string;
   spanId: string;
-  parentSpanId: string;
+  parentSpanId?: string;
```

Comment:
Why did this field become optional?

Reply:
To match the [API data model](https://cloud.google.com/trace/docs/reference/v1/rest/v1/projects.traces#TraceSpan) more closely. This field is never read except in tests so I don't think it really makes a difference.